covalent/executor/executor_plugins/dask.py

```diff
@@ -66,13 +68,11 @@ def __init__(
         log_stderr: str = "stderr.log",
         conda_env: str = "",
         cache_dir: str = "",
+        workdir: str = "",
         current_env_on_conda_fail: bool = False,
     ) -> None:
```

Comment:
```suggestion
        current_env_on_conda_fail: bool = False,
        workdir: str = "",
    ) -> None:
```
Same reason as earlier.

Reply:
Done!